forem/forem
# Replace honeybadger-js with @honeybadger-io/js

## What type of PR is this? (check all applicable)

## Description
My previous PR https://github.com/forem/forem/pull/15728 did not work. It turns out I was using a syntax that belongs to a newer version of Honeybadger, which is pointing to a different repository. This PR upgrades to it by following [their upgrade doc](https://docs.honeybadger.io/lib/javascript/support/upgrading-to-v3/).

## Related Tickets & Documents
https://docs.honeybadger.io/lib/javascript/support/upgrading-to-v3/
## QA Instructions, Screenshots, Recordings
HoneybadgerJS is disabled by default in development but there are two quick test to verify it's working.
- Make sure you were able to load up local development without issue.
- With a browser console of your choice, running `Honeybadger.notify("ping")` will return `false`, which means it's running fine.

## Added tests?

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
n/a
